io/src/main/java/org/commonjava/maven/ext/io/FileIO.java

```diff
@@ -16,23 +16,42 @@
 package org.commonjava.maven.ext.io;
 
 import org.apache.commons.io.FileUtils;
+import org.commonjava.maven.ext.common.ManipulationException;
 import org.commonjava.maven.ext.io.resolver.GalleyInfrastructure;
+import org.jdom2.output.LineSeparator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import javax.inject.Inject;
 import javax.inject.Named;
 import javax.inject.Singleton;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
 import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
 import java.net.URL;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
 import java.util.UUID;
 
+import com.google.common.primitives.Chars;
```

Comment:
Where is this from/used?

Reply:
This is just a case of the IDE importing something while typing (probably wanted `StandardCharsets`). I wish there was just a way it would remove any unused imports on save, but it can be removed manually.

Reply:
Ok, lets remove it